usaspending_api/agency/v2/views/program_activity_list.py

```diff
@@ -1,14 +1,14 @@
-from django.db.models import Sum, F
+from django.db.models import Sum, F, Q
 from rest_framework.request import Request
 from rest_framework.response import Response
 from typing import Any, List
-from usaspending_api.agency.v2.views.agency_base import AgencyBase
+from usaspending_api.agency.v2.views.agency_base import AgencyBase, ListMixin
 from usaspending_api.common.cache_decorator import cache_response
 from usaspending_api.common.helpers.generic_helper import get_simple_pagination_metadata
 from usaspending_api.references.models import RefProgramActivity
 
 
-class ProgramActivityList(AgencyBase):
+class ProgramActivityList(AgencyBase, ListMixin):
```

Comment:
Mixins are typically listed to the left of base classes in python because method overrides are set up based on a right to left reading of the superclasses (i.e. if you want mixin methods to override base class methods, they need to be listed to the left of the base class).  It's not currently an issue in this case since there are no overrides, but if someone decides to override a base method/property this will become a problem.